app/components/npq_separation/admin/statement_selector_component.html.erb

```diff
@@ -1,3 +1,4 @@
+<div class="govuk-grid-row">
```

Comment:
Normally you'd have a `columns` div inside the row div? Otherwise, why have the row div?